addons/calendar/tests/test_event_notifications.py

```diff
@@ -201,26 +200,32 @@ def test_notification_event_timezone(self):
         """
             Check the domain that decides when calendar events should be notified to the user.
         """
-        def search_event():
+        def search_events():
```

Comment:
Hello :wave: 
I'm not sure if changing the method name in test files in stable is allowed, or if it is forbidden just like in regular classes.
What do you think? @arj-odoo

Reply:
@nle-odoo do you know if this is ok in stable?

Reply:
It brings no value to change it in stable. It could be monkey patched. Just update that name in the last PR of the FW port.